webpack.config.search.js

```diff
@@ -0,0 +1,23 @@
+const path = require( 'path' );
+
+module.exports = {
+	entry: path.join( __dirname, '_inc/search/src/index.jsx' ),
+	output: {
+		path: path.resolve( __dirname, '_inc/search/dist' ),
+		filename: 'jp-search.bundle.js',
+	},
+	module: {
+		rules: [
+			{
+				test: /\.(js|jsx)$/,
+				exclude: /node_modules/,
+				use: {
+					loader: 'babel-loader',
+				},
+			},
+		],
+	},
+	resolve: {
+		extensions: [ '.js', '.jsx' ],
+	},
+};
```

Comment:
You might not need this extra Webpack config if you;
- add new entry points either to the main Webpack file (`webpack.config.js`) or the one geared towards Gutenberg extensions (`webpack.config.extensions.js`).
- Use Calypso-build directly from the command line (see [docs](https://github.com/Automattic/wp-calypso/tree/master/packages/calypso-build#usage) so that you won't need config.

Ideally there would be only one build command and one Webpack config. I still hope that sooner than later the two files would become one. :-)

Reply:
I went ahead and merged the relevant parts of `webpack.config.search.js` into `webpack.config.js`.

Reply:
See my related question about block editor extension: https://github.com/Automattic/jetpack/pull/12741#issuecomment-507424788